### ISSUE TITLE
Testing for Brave Together

### ISSUE THREAD

**Issue by mrose17** [brave/qa-resources#188]
#### Release Date/Target:

* Release Date: **`Pending`**

#### Summary:

@mrose17 has requested QA resources for Brave Today. However, still waiting on information re: when this feature will land/get released. Believe there's still some work pending from @mrose17.

* **`Issue:`** - https://github.com/brave/internal/issues/727
* **`GH`** - https://github.com/brave/brave-together

**Comment by stephendonner** [comment]
Sorry for the delay; it's up in "working draft" mode and I'm iterating as quickly and accurately as I can, while also testing/working out issues 👍 

https://github.com/brave/qa-resources/wiki/Brave-Talk-Test-Plan-(MVP)

**Comment by kjozwiak** [comment]
Closing this off as we've released `Brave Talk`. @stephendonner please create a new issue if there's any other work that we might need to track via the our project board.
